src/pages/Facility/billing/invoice/InvoiceShow.tsx

```
@@ -353,969 +363,1012 @@ export function InvoiceShow({
+        <div className="flex items-start justify-between flex-col sm:flex-row gap-4 sm:items-center border-b-3 border-double pb-4">
+          <div className="flex gap-3 sm:gap-6 flex-col md:flex-row">
+            <BackButton variant="link" className="px-0 justify-start">
+              <ChevronLeft />
+              <span>{t("back")}</span>
+            </BackButton>
+            <div className="h-auto w-px bg-gray-300" aria-hidden="true" />
             <div>
               <label className="text-gray-700 text-sm font-medium">
-                {t("amount_due")}
+                {t("patient_name")}
               </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)